plasmapy/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import helper
```

Comment:
(*low hanging fruit* 🤣) I feel this sub-package would be more appropriately named as `plasmapy.tests.helpers`. There's more than one helper contained here.

Reply:
Yeah, I was following Astropy's lead since they have `astropy.test.helper`.  I personally would prefer `plasmapy.test.helpers` so we can go with that. :joy_cat:

Reply:
Or, we could spend another 2 hrs in the next meeting debating the singular vs plural. 🤣